context/app/static/js/components/entity-search/SearchWrapper/store.js

```diff
@@ -5,16 +5,18 @@ import createContext from 'zustand/context';
 
 const { Provider, useStore } = createContext();
 
-const createStore = ({ initialFields, initialFacets, fields, facets, filters, entityType }) =>
+const createStore = ({ initialFields, initialFacets, fields, facets, filters, entityType, numericFacetsProps }) =>
   create((set) => ({
     initialFields,
     initialFacets,
     fields,
     facets,
     filters,
     entityType,
+    numericFacetsProps,
     setFields: (selectedFields) => set({ fields: selectedFields }),
     setFacets: (selectedFacets) => set({ facets: selectedFacets }),
+    setNumericFacetsProps: (numericFacets) => set({ numericFacetsProps: numericFacets }),
```

Comment:
As the list gets longer, would it make sense to pair the setters and the things they set? Or if this is more conventional, ignore.